tests/backends/test_ddp.py

```diff
@@ -23,6 +23,9 @@
 ])
 @pytest.mark.skipif(torch.cuda.device_count() < 2, reason="test requires multi-GPU machine")
 def test_multi_gpu_model_ddp_fit_only(tmpdir, cli_args):
+
+    os.environ["PL_ENABLE_DDP_COVERAGE"] = '1'
```

Comment:
maybe add this env setting to the file head because it shall be done for all tests here?